src/pybind/mgr/prometheus/module.py

```diff
@@ -108,6 +108,13 @@ def health_status_to_number(status):
 
 NUM_OBJECTS = ['degraded', 'misplaced', 'unfound']
 
+HEALTH_CHECKS = [
+    {
+        "name": 'SLOW_OPS',
+        "description": 'OSD requests taking a long time to process',
```

Comment:
this is not accurate. monitor also reports SLOW_OPS health checks. requests targeting monitor pending for long time are considered SLOW_OPS.

Reply:
This matches the documentation in https://docs.ceph.com/en/latest/rados/operations/health-checks/. Whilst there are other conditions that SLOW_OPS may trigger on, OSD is the most frequent in my experience anyway.

Reply:
i insist. i was involved in couple customer issues where an obvious symptom was slow mon ops.

Reply:
Please suggest an alternate text and I'll add it. As I said, I've used current docs - but experience trumps docs :)

Reply:
i created #37832 to update the related document.

> Please suggest an alternate text and I'll add it. 

```py
"description": 'OSD or monitor requests taking a long time to process',
```

Reply:
Thanks.